client/extensions/woocommerce/app/settings/taxes/index.js

```diff
@@ -0,0 +1,203 @@
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Main from 'components/main';
+import { successNotice, errorNotice } from 'state/notices/actions';
+import TaxesNexus from './taxes-nexus';
+import TaxesOptions from './taxes-options';
+import TaxesRates from './taxes-rates';
+
+class SettingsTaxes extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			isSaving: false,
+			pricesIncludeTaxes: true,
+			shippingIsTaxable: true,
+			taxesEnabled: true,
+			userBeganEditing: false,
+		};
+	}
+
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string,
+		} ),
+		className: PropTypes.string,
+	};
+
+	componentDidMount = () => {
+		const { site } = this.props;
+
+		if ( site && site.ID ) {
+			this.props.fetchSettingsGeneral( site.ID );
+			this.props.fetchTaxSettings( site.ID );
+		}
+	}
+
+	componentWillReceiveProps = ( newProps ) => {
+		if ( ! this.state.userBeganEditing ) {
+			this.setState( {
+				pricesIncludeTaxes: newProps.pricesIncludeTaxes,
+				shippingIsTaxable: newProps.shippingIsTaxable,
+				taxesEnabled: newProps.taxesEnabled,
+			} );
+		}
+	}
+
+	onEnabledChange = () => {
+		this.setState( { taxesEnabled: ! this.state.taxesEnabled, userBeganEditing: true } );
+	}
+
+	onCheckboxChange = ( event ) => {
+		const option = event.target.name;
+		const value = event.target.checked;
+		this.setState( { [ option ]: value, userBeganEditing: true } );
+	}
+
+	pageHasChanges = () => {
+		return this.state.userBeganEditing;
+	}
+
+	onSave = ( event ) => {
+		const { site, translate } = this.props;
+
+		event.preventDefault();
+		this.setState( { isSaving: true } );
+
+		const onSuccess = () => {
+			this.setState( { isSaving: false, userBeganEditing: false } );
+			return successNotice( translate( 'Settings updated successfully.' ) );
+		};
+
+		const onFailure = () => {
+			this.setState( { isSaving: false } );
+			return errorNotice( translate( 'There was a problem saving your changes. Please try again.' ) );
+		};
+
+		// TODO - chain these
+
+		this.props.updateTaxesEnabledSetting(
+			site.ID,
+			this.state.taxesEnabled,
+			null,
+			null,
+		);
+
+		this.props.updateTaxSettings(
+			site.ID,
+			this.state.pricesIncludeTax,
+			! this.state.shippingIsTaxable, // note the inversion
+			onSuccess,
+			onFailure
+		);
+	};
+
+	render = () => {
+		const { className, loading, site, translate } = this.props;
+
+		if ( loading ) {
+			// TODO placeholder
+			return null;
+		}
+
+		const breadcrumbs = [
+			( <a href={ getLink( '/store/:site/', site ) }>{ translate( 'Settings' ) }</a> ),
+			( <span>{ translate( 'Taxes' ) }</span> ),
+		];
+
+		const saveButtonDisabled = this.state.isSaving || ! this.pageHasChanges();
+
+		return (
+			<Main className={ classNames( 'settings-taxes', className ) }>
+				<ActionHeader breadcrumbs={ breadcrumbs }>
+					<Button disabled={ saveButtonDisabled } onClick={ this.onSave } primary >
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15541/commits/17f50964f1f6156641032d53a6b96785ad2895a0